### DISCUSSION TITLE
Absolute setup with Relative cursor

### DISCUSSION THREAD

**Discussion by NotRaffi** [OpenTabletDriver/OpenTabletDriver#1383] [Ideas]
Hey, this might be a stretch, but I think the Relative setup cursor is amazing, with the cursor not jumping across the screen and staying in place, even though you lift the cursor and starts at a different location in the tablet, but it's pretty hard to setup.

Opposite of that, Absolute setup cursor is pretty hard to use [I'm sorry if it's just me I'm bad at remembering places to click using this Setup because the cursor directly corresponds to the tablet] but the setup is easy to use

I hope this makes sense, here is the TL;DR: 

Absolute:
Easy to set, confusing cursor

Relative:
Hard to set, simple cursor

**Comment by InfinityGhost** [comment]
We haven't figured out a good way to display relative position changes in the way we have absolute position, at least not in an intuitive way. If you have any suggestions, shoot.
